Examples/Python/python/acts/_adapter.py

```diff
@@ -110,14 +110,12 @@ def create(*args, mdecorator=None, **kwargs):
             cfg = cls.Config()
         else:
             cfg = config_class()
-        _kwargs = {}
         for k, v in kwargs.items():
-            try:
-                setattr(cfg, k, v)
-            except AttributeError:
-                _kwargs[k] = v
-        det = cls()
-        tg, deco = det.finalize(cfg, mdecorator, *args, **_kwargs)
+            setattr(cfg, k, v)
+        if hasattr(cfg, "materialDecorator"):
+            setattr(cfg, "materialDecorator", mdecorator)
+        det = cls(cfg)
+        tg, deco, _ = det.trackingGeometry()
```

Comment:
Do we even need this create function now?

Reply:
I think this can be killed now